packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -238,12 +240,7 @@ class IOSSimulator extends Device {
     }
   }
 
-  @override
-  bool isConnected() {
-    if (!Platform.isMacOS)
-      return false;
-    return SimControl.instance.getConnectedDevices().any((SimDevice device) => device.udid == id);
-  }
+  bool isConnected() => Platform.isMacOS;
```

Comment:
Actually, I think I'll remove the method in a follow-on PR. It doesn't really make sense any more. We used to create devices w/o knowing whether they existed - we don't do that anymore.